script.js

```diff
@@ -0,0 +1,124 @@
+            if (demetrius_follow === "S" || demetrius_follow === "s") {
+                var demetrius_serenade = prompt("SERENADE HELENA\nYou try to convince Helena to choose you over Lysander, but she thinks you are mocking her, as neither of you loved her before.\nAfter a hilarious quarrel among the four lovers, you all become too weary and fall asleep. However, when you wake up, you are pleased to find Helena sleeping beside you and later learn that Lysander is in love with Hermia again. It turns out that this mishap was all of the work of fairies.\nThe four of you return home, have a joint wedding, and live happily ever after. Congratulations!");
+            } else if (demetrius_follow === "D" || demetrius_follow === "d") {
+                var demetrius_duel = prompt("DUEL LYSANDER\nLysander accepts your challenge, but as the two of you run off to find a good location, you suddenly become very tired and immediately fall asleep.\nWhen you wake up, you are pleased to find Helena sleeping beside you and later learn that Lysander is in love with Hermia again. It turns out that this mishap was all of the work of fairies.\nThe four of you return home, have a joint wedding, and live happily ever after. Congratulations!");
+            }
+    } else if (demetrius_intro === "B" || demetrius_intro === "b") {
+        var demetrius_bachelor = prompt("REMAIN A BACHELOR\nI forgot to mention that you are poor and that marrying into Hermia's family will bring you riches. Do you want to change your previous answer (Y/N)?");
+            if (demetrius_bachelor === "Y" || demetrius_bachelor === "y") {
+                var demetrius_yes = prompt("Good choice. You follow the pair of lovers into the forest, and it turns out that Helena has followed you and pleads for your love. You refuse.\nHowever, something strange happens while you are asleep and you suddenly are in love with Helena! BUT Lysander also loves Helena now too!\nYou challenge Lysander to a duel. He accepts your challenge, but as the two of you run off to find a good location, you suddenly become very tired and immediately fall asleep.\nWhen you wake up, you are pleased to find Helena sleeping beside you and later learn that Lysander is in love with Hermia again. It turns out that this mishap was all of the work of fairies.\nThe four of you return home, have a joint wedding, and live happily ever after. Congratulations!");
+            } else if (demetrius_bachelor === "N" || demetrius_bachelor === "n") {
+                var demetrius_no = prompt("That annoying Helena still follows you around, and you remain poor and alone for the rest of your days.");
+            }
+    }
+}
+}
+//Other numbers.
```

Comment:
It's fine to comment out some code while you are quickly testing something, but just make sure to clean it up right away to practice good habit :)